tests/StringValidatorTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Brick\Validation\Tests;
+
+use Brick\Validation\Validator\StringValidator;
+
+/**
+ * Unit tests for the String validator.
+ */
+class StringValidatorTest extends AbstractTestCase
+{
+    public function testStringValidator()
+    {
+        $validator = new StringValidator();
+
+        $this->doTestValidator($validator, [
+            ''   => [],
+            null => [],
```

Comment:
`null` is [casted to a string](https://3v4l.org/JQ2XC) in this context, so you can effectively remove this line! Plus, the key is passed as a parameter to [isValid()](https://github.com/brick/validation/blob/master/src/Validator.php#L19), which only accepts a string.

Reply:
Ok, I remove the ```null``` key.